src/Rubric/RubSmalltalkEditor.class.st

```diff
@@ -450,13 +450,14 @@ RubSmalltalkEditor >> debug: aCompiledMethod receiver: anObject in: evalContext
 
 	| process suspendedContext |
 	process := [ 
-	           aCompiledMethod
-		           valueWithReceiver: anObject
-		           arguments:
-		           (evalContext
-			            ifNil: [ #(  ) ]
-			            ifNotNil: [ { evalContext } ]) ] newProcess.
+		aCompiledMethod
+			valueWithReceiver: anObject
+			arguments: (aCompiledMethod numArgs = 0 
+				ifTrue: [ #() ] 
+				ifFalse: [ { evalContext } ] ) ]
+			newProcess.
```

Comment:
This was missing fix to be compatible with DoIt without arguments (no ThisContext now)